adcc/backends/pyscf.py

```diff
@@ -62,6 +64,63 @@ def electric_dipole_velocity(self):
                 self.scfres.mol.intor('int1e_ipovlp', comp=3, hermi=2)
             )
 
+    @property
+    def electric_quadrupole(self):
+        def g_origin_dep_ints_el_quad(gauge_origin):
+            gauge_origin = _determine_gauge_origin(self.scfres, gauge_origin)
+            with self.scfres.mol.with_common_orig(gauge_origin):
+                r_r = self.scfres.mol.intor_symmetric('int1e_rr', comp=9)
```

Comment:
I think, the minus sign for the negative charge is missing here, right? Also for the other quadrupole operators. And could you include docstrings specifying the implemented operator integrals similar to the ones above?